param/low_level_motion_sim.yaml

```diff
@@ -19,26 +19,22 @@ roll_d: 0.0
 roll_accumulator_max: 2.0
 roll_accumulator_min: -2.0
 
-yaw_p: -1.0
-yaw_i: 0.0
-yaw_d: 0.0
-yaw_accumulator_max: 0.0
-yaw_accumulator_min: 0.0
-
 throttle_max      : 1.0
 throttle_min      : 0.0
 throttle_max_rate : 1000.0
 
-pitch_max         : 3.5
-pitch_min         : -3.5
-pitch_max_rate    : 1000.0
+pitch_max         : 0.25
```

Comment:
I don't think we should turn the max limits or the rate limits back on. They severely affected the stability of the aircraft in the simulator. I think we should have a good discussion about them soon.